addons/mail/static/src/messaging/component/follower_subtype_list/follower_subtype_list.js

```diff
@@ -0,0 +1,81 @@
+odoo.define('mail.messaging.component.FollowerSubtypeList', function (require) {
+'use strict';
+
+const components = {
+    FollowerSubtype: require('mail.messaging.component.FollowerSubtype'),
+};
+const useStore = require('mail.messaging.component_hook.useStore');
+
+const { Component, QWeb } = owl;
+
+class FollowerSubtypeList extends Component {
+
+    /**
+     * @override
+     */
+    constructor(...args) {
+        super(...args);
+        useStore(props => {
+            const followerSubtypeList = this.env.entities.FollowerSubtypeList.get(props.localId);
+            const follower = followerSubtypeList
+                ? followerSubtypeList.follower
+                : undefined;
+            const followerSubtypes = follower ? follower.subtypes : [];
```

Comment:
Object/array created (implicitly or explicitly) inside `useStore` need `compareDepth` to be set to avoid excessive renders (because the compared object itself is different every time).

Same applies to all getters/relations that do `map` or create sub objects or arrays indirectly.